.travis.yml

```diff
@@ -29,4 +30,5 @@ after_success:
   - 'curl -Lo travis_after_all.py https://git.io/travis_after_all'
```

Comment:
you can get rid of these 3 lines while you're at it. They were only needed in very old versions of semantic release